### PR TITLE
Fix/python3

### LINKED ISSUE #17: `chalmers @startup` doesn't work on Python 3
```
C:\Windows\system32>chalmers @startup enable
Your password is required by the windows service manager to launch the chalmers
service at startup
[ERROR]
Traceback (most recent call last):
  File "c:\conda64\Scripts\chalmers-script.py", line 5, in <module>
    sys.exit(chalmers.scripts.chalmers_main.main())
  File "c:\conda64\lib\site-packages\chalmers\scripts\chalmers_main.py", line 36
, in main
    run_command(args, exit=exit)
  File "c:\conda64\lib\site-packages\clyent\__init__.py", line 151, in run_comma
nd
    return args.main(args)
  File "c:\conda64\lib\site-packages\chalmers\commands\at_startup.py", line 50,
in main
    service.install()
  File "c:\conda64\lib\site-packages\chalmers\service\win32_system_service.py",
line 48, in install
    password = getpass.getpass(b"Password for %s: " % self.target_user)
TypeError: %b requires bytes, or an object that implements __bytes__, not 'str'
```

### PR DESCRIPTION
**Pull request by srossross**
fixes #17

### CODE REVIEW

File: chalmers/service/win32_system_service.py [hunk 1]
```diff
@@ -56,32 +56,32 @@ def uninstall(self):
...
 
         if is_running(self.target_user):
-            log.info("Service is running, stopping service %s" % self.service_name)
+            log.info("Service is running, stopping service {}".format(self.service_name))
             StopService(self.service_name)
 
         if is_installed(self.target_user):
             RemoveService(self.service_name)
-            log.info("Uninstalled windows service '%s'" % self.service_name)
+            log.info("Uninstalled windows service '%s'".format(self.service_name))
```
Comment (bollwyvl) [inline chalmers/service/win32_system_service.py]: Should be '{}'?

File: chalmers/service/win32_system_service.py [hunk 2]
```diff
@@ -56,32 +56,32 @@ def uninstall(self):
...
         else:
-            log.error("Windows service '%s' is not installed" % self.service_name)
+            log.error("Windows service '%s' is not installed".format(self.service_name))
 
     def status(self):
 
-        log.info("Status for service '%s'" % self.service_name)
+        log.info("Status for service '%s'".format(self.service_name))
```
Comment (bollwyvl) [inline chalmers/service/win32_system_service.py]: Should be '{}'?

File: chalmers/service/win32_system_service.py [hunk 3]
```diff
@@ -56,32 +56,32 @@ def uninstall(self):
...
+            log.info("Service is running, stopping service {}".format(self.service_name))
             StopService(self.service_name)
 
         if is_installed(self.target_user):
             RemoveService(self.service_name)
-            log.info("Uninstalled windows service '%s'" % self.service_name)
+            log.info("Uninstalled windows service '%s'".format(self.service_name))
         else:
-            log.error("Windows service '%s' is not installed" % self.service_name)
+            log.error("Windows service '%s' is not installed".format(self.service_name))
```
Comment (bollwyvl) [inline chalmers/service/win32_system_service.py]: Should be '{}'?

File: chalmers/service/win32_system_service.py (hunk 2 above)
Comment (srossross) [inline chalmers/service/win32_system_service.py]: Thanks @bollwyvl . fixed in master.

**Pull request merged**